README-zh_CN.md

```diff
@@ -123,7 +123,7 @@ Fluid让Kubernetes真正具有分布式数据缓存的基础能力，开源只
 
 钉钉讨论群
 <div>
-  <img src="http://kubeflow.oss-cn-beijing.aliyuncs.com/Static/dingtalk.png" width="280" title="dingtalk">
+  <img src="static/dingtalk.png" width="280" title="dingtalk">
```

Comment:
Please update to the latest jpg? I will send the picture to you.

Reply:
done.